### PR TITLE
Link buttons[0] and buttons[1] to primary actions

### PR DESCRIPTION
**Pull request by Manishearth**
Requires https://github.com/immersive-web/webxr/pull/893

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -209,7 +211,7 @@ This module extends the {{GamepadMappingType}} to describe the mapping of common
...
+In order to report a {{Gamepad/mapping}} of {{GamepadMappingType/"xr-standard"}} the device MUST report a {{XRInputSource/targetRayMode}} of {{XRTargetRayMode/"tracked-pointer"}} and MUST have a non-<code>null</code> {{XRInputSource/gripSpace}}. It MUST have at least one <dfn>primary button</dfn>, often a trigger, separate from any touchpads or thumbsticks. The [=primary button=] MUST trigger the [=primary action=] for the input source. The device MAY have a <dfn>primary squeeze button</dfn>, which MUST trigger the [=primary squeeze action=] for the input source. If a device does not meet the requirements for the {{GamepadMappingType/"xr-standard"}} mapping it may still expose a {{XRInputSource/gamepad}} with a {{Gamepad/mapping}} of <code>""</code> (empty string). The {{GamepadMappingType/"xr-standard"}} mapping MUST only be used by {{Gamepad}} instances reported by an {{XRInputSource}}.
```
Comment (NellWaliczek) [inline index.bs]: > have at least one <dfn>primary button</dfn>, often a trigger

In #17 it appears we settled on this needing to be a trigger.  Mind fixing that at the same time?

Comment (NellWaliczek) [inline index.bs]: nit: might be clearer to say?
> The device MAY have a <dfn>primary squeeze button</dfn>, which, if present, MUST trigger the [=primary squeeze action=] for the input source.

**Comment by Manishearth** [comment]
Addressed

**Comment by Manishearth** [comment]
Actually, there's no need to keep this a draft, as long as folks know this is contingent on immersive-web/webxr#893

**Pull request merged**